lldb/unittests/Host/posix/SupportTest.cpp

```diff
@@ -7,21 +7,24 @@
 //===----------------------------------------------------------------------===//
 
 #include "lldb/Host/linux/Support.h"
+#include "lldb/Host/posix/Support.h"
 #include "llvm/Support/Threading.h"
 #include "gtest/gtest.h"
 
 using namespace lldb_private;
 
 TEST(Support, getProcFile_Pid) {
```

Comment:
And then make this `#ifndef __APPLE__`, as that's the only OS which definitely doesn't have proc. It looks like the BSD should mostly have them. If it turns out to be  a problem, we opt them out as well.